AUTHORS

```diff
@@ -1050,3 +1050,50 @@ Rodrigo Luger <rodluger@gmail.com>
 Marco Antônio Habitzreuter <mahabitzreuter@gmail.com>
 Ayush Bisht <bisht.ayush2001@gmail.com>
 Akshansh Bhatt <akshansh@tuta.io>
+Brandon T. Willard <brandonwillard@users.noreply.github.com>
+Thomas Aarholt <thomasaarholt@gmail.com>
+Hiren Chalodiya <hirenchalodiya99@gmail.com>
+Roland Dixon <rols121@gmail.com>
+dimasvq <dimas.vq.2020@bristol.ac.uk>
+Sagar231 <sagarfeb298@gmail.com>
+Michael Chu <michael02chu@gmail.com>
+Abby Ng <abigailjng@gmail.com>
+Angad Sandhu <55819847+angadsinghsandhu@users.noreply.github.com>
+Alexander Cockburn <alexander_cockburn12@hotmail.com>
+Yaser <yaseralosh@outlook.com>
+Davide_sd <sandona.davide@gmail.com>
+gutow <gutow@uwosh.edu>
```

Comment:
I've emailed asking what name to use

Reply:
I replied to your e-mail, but will also put it here. Please use `Jonathan Gutow <gutow@uwosh.edu>`. Thanks.

Reply:
This should be
```
Jonathan Gutow <gutow@uwosh.edu>
```